gitkraken-client/pull-requests.md

```diff
@@ -87,9 +92,19 @@ From this view, GitHub users may edit the pull request:
 - Milestones
 - Labels
 
-From the upper right of the Pull Request view, you may click the <button class='button button--primary button--ui button--nolink'><span style='color:#141422;'>View file changes</span></button> button to review the affected files for this pull request. Note, code review and code comment are not currently available from within GitKraken Client.
 
-<img src='/wp-content/uploads//view-changes.png' srcset='/wp-content/uploads//view-changes@2x.png' class='img-bordered img-responsive center'>
+From the upper right of the Pull Request view, you may click the <button class='button button--primary button--ui button--nolink'><span style='color:#141422;'>Review Code and Suggest Changes</span></button> button to review the affected files for this pull request. Note, code review and code comment are not currently available from within GitKraken Desktop.
+
+<img src='/wp-content/uploads/gkc-pr-code-suggestions.png' class='img-bordered img-responsive center'>
+
+
+### Review Code and Suggest Changes
+
+In Gitkraken Desktop, Review Code and Suggest Changes simplifies code review by allowing you to make suggestions and edits across the entire project, not just on the lines that were changed, GitKraken Desktop, and gitkraken.dev. When a Pull Request is open, you can make suggestions to the pull request that others can then review and accept to include in the pull request. 
```

Comment:
> changed, GitKraken Desktop, and gitkraken.dev.

to 

> changed, in GitKraken Desktop, GitLens, and gitkraken.dev.

---

Also, we should include a section about the recipient opening and accepting the change